### PR TITLE
danish month abbrevations should end with period

### PR DESCRIPTION
**Pull request by rasck**
Fixed abbreviations on danish months. All but "maj" (May) should end with period. 

I tried to run
`"locale-snapshots": "env TZ=utc babel-node ./scripts/build/localeSnapshots/index.js",`

But i'm sitting on a windows machine and the `env TZ=utc` doesn't seem to work, so i would mess with all the snapshots...

### CODE REVIEW

**Comment by kossnocorp** [comment]
Wow, I had no idea about the issue with `TZ` on Windows. I'm wondering how did we managed to avoid it so far. Anyway, I've updated the snapshot for you 👌

File: src/locale/da/snapshot.md [hunk 1]
```diff
@@ -24,16 +24,16 @@
 |                                 |              | 2019-04-01T12:13:14.015Z | 2. kvartal                                         | 2019-04-01T00:00:00.000Z |
 | Month (formatting)              | Mo           | 2019-02-11T12:13:14.015Z | 2.                                                 | 2019-02-01T00:00:00.000Z |
 |                                 |              | 2019-07-10T12:13:14.015Z | 7.                                                 | 2019-07-01T00:00:00.000Z |
-|                                 | MMM          | 2019-02-11T12:13:14.015Z | feb                                                | 2019-02-01T00:00:00.000Z |
-|                                 |              | 2019-07-10T12:13:14.015Z | jul                                                | 2019-07-01T00:00:00.000Z |
+|                                 | MMM          | 2019-02-11T12:13:14.015Z | feb.                                               | Invalid Date             |
+|                                 |              | 2019-07-10T12:13:14.015Z | jul.                                               | Invalid Date             |
```
Comment (kossnocorp) [inline src/locale/da/snapshot.md:28]: As you can see, the change broke `parse`. Please add dots to the corresponding match patterns: https://github.com/date-fns/date-fns/blob/master/src/locale/da/_lib/match/index.js#L27

**Comment by kossnocorp** [comment]
No worries, just ping me when you're done and I'll rebuild it again.

Regarding the Dutch locale, I guess I've missed the problem, yes, please fix it as well 🙏

**Comment by rasck** [comment]
@kossnocorp done 🙂

**Pull request merged**